tools/wpt/browser.py

```diff
@@ -644,10 +643,21 @@ def find_webdriver(self, channel=None):
     def install_webdriver(self, dest=None, channel=None):
         raise NotImplementedError
 
-    def version(self, binary):
+    def version(self, binary=None, webdriver_binary=None):
+        if webdriver_binary is not None:
+            try:
+                output = call(webdriver_binary, "--version").strip()
+                # TODO: explain
+                if output.startswith('Included with '):
+                    output = output[14:]
+                return output
+            except subprocess.CalledProcessError:
+                # TODO stderr gets printed :'(
+                pass
+        # TODO: this message doesn't show up anywhere
```

Comment:
I don't know why this, is the logger not wired up correctly?

Reply:
I think the logger handler gets wired up late, but we should probably fix that.